CONTRIBUTING.md

```diff
@@ -17,11 +17,12 @@ pnpm install
 pnpm build
 ```
 
-You can now run SvelteKit by linking it into your project with [pnpm `overrides`](https://pnpm.io/package_json#pnpmoverrides) as demonstrated in the [sandbox example](https://github.com/sveltejs/kit-sandbox) or by running one of the test projects as described in [the testing section](#testing) below.
+You can now run SvelteKit by linking it into your project with [pnpm `overrides`](https://pnpm.io/package_json#pnpmoverrides) as demonstrated in the [sandbox example](https://github.com/sveltejs/kit-sandbox/blob/main/package.json) or by running one of the test projects as described in [the testing section](#testing) below.
```

Comment:
I actually think linking to the readme might be better because it gives a lot of explanation about those particular lines in the `package.json`, how to set it up, etc.
```suggestion
You can now run SvelteKit by linking it into your project with [pnpm `overrides`](https://pnpm.io/package_json#pnpmoverrides) as demonstrated in the [sandbox example](https://github.com/sveltejs/kit-sandbox) or by running one of the test projects as described in [the testing section](#testing) below.
```

Reply:
Ah make sense. I didn't notice the readme 🙈